cpp/modmesh/toggle/profile.hpp

```diff
@@ -145,13 +155,12 @@ class TimeRegistry
     std::string report() const
     {
         std::ostringstream ostm;
+        ostm << "      " << total_call_count() << " function calls in " << total_time() << " seconds" << std::endl;
+        ostm << std::endl;
+        ostm << std::setw(40) << "Function Name" << std::setw(25) << "Call Count" << std::setw(25) << "Total Time (s)" << std::setw(25) << "Per Call (s)" << std::setw(25) << "Cumulative Time (s)" << std::setw(25) << "Per Call (s)" << std::endl;
         for (auto it = m_entry.begin(); it != m_entry.end(); ++it)
         {
-            ostm
-                << it->first << " : "
-                << "count = " << it->second.count() << " , "
-                << "time = " << it->second.time() << " (second)"
-                << std::endl;
+            ostm << std::setw(40) << it->first << std::setw(25) << it->second.count() << std::setw(25) << it->second.time() << std::setw(25) << it->second.time() / it->second.count() << std::setw(25) << it->second.ctime() << std::setw(25) << it->second.ctime() / it->second.count() << std::endl;
         }
         return ostm.str();
     }
```

Comment:
It seems that enhancement of the report function make the CI fail.
I will fix the modification later.